documentation/library-reference/source/dylan/threads.rst

```diff
@@ -748,7 +748,7 @@ Locks
      *with-lock* may signal a condition of the following class (a
```

Comment:
This too should be linked or at least backticks instead of asterisks...

Reply:
I added another commit to https://github.com/dylan-lang/opendylan/pull/1495 to get this. Thanks!